src/observability/summarizer.go

```diff
@@ -60,6 +75,18 @@ func extractNetworkInfoFromSystemLog(netLog pb.Log) (string, string, string, str
 			if strings.Contains(locres, "sin_addr") {
 				bindAddress = strings.Split(locres, "=")[1]
 			}
+			if strings.Contains(locres, "sa_family") {
+				protocol = strings.Split(locres, "=")[1]
+				if protocol == "AF_INET6" {
+					protocol = strings.Replace(protocol, "AF_INET6", "TCPv6", 1)
+				} else if protocol == "AF_NETLINK" {
+					protocol = strings.Replace(protocol, "AF_NETLINK", "NETLINK", 1)
+				} else if protocol == "AF_INET" {
+					protocol = strings.Replace(protocol, "AF_INET", "TCP", 1)
+				} else if protocol == "AF_UNIX" {
+					protocol = strings.Replace(protocol, "AF_UNIX", "UNIX", 1)
+				}
+			}
```

Comment:
```suggestion
			if strings.Contains(locres, "sa_family") {
				protocol = strings.Split(locres, "=")[1]
			}
```

Reply:
It is better to show protocol name as it is. Also there are more types of AF_* families out there which are not considered in this condition. If any of it shows up the value will be empty.

Reply:
Thank you for reviewing. Changes are done. Now table will look like this.
```

Bind Points
+------------+---------------------------+-----------+--------------+-------+------------------------------+
|  PROTOCOL  |          COMMAND          | BIND PORT | BIND ADDRESS | COUNT |      LAST UPDATED TIME       |
+------------+---------------------------+-----------+--------------+-------+------------------------------+
| AF_NETLINK | /usr/lib/apt/methods/http |           |              | 2     | Thu Dec 15 15:46:41 IST 2022 |
| AF_INET    | /bin/nc.traditional       | 4444      | 0.0.0.0      | 1     | Thu Dec 15 15:48:03 IST 2022 |
+------------+---------------------------+-----------+--------------+-------+------------------------------+
```